.circleci/config.yml

```diff
@@ -18,8 +18,66 @@ jobs:
           PATH=$HOME/miniconda3/envs/pandas-dev/bin:$HOME/miniconda3/condabin:$PATH
           LD_PRELOAD=$HOME/miniconda3/envs/pandas-dev/lib/libgomp.so.1:$LD_PRELOAD
           ci/run_tests.sh
+  build-aarch64:
+    parameters:
+      cibw-build:
+        type: string
+    machine:
+      image: ubuntu-2004:202101-01

```

Comment:
Does it make sense to bump to `ubuntu-2004:2022.04.1`?

Reply:
I'll do it in a follow up. I'm currently matching the other arm64 job.